### DISCUSSION TITLE
3.7.2无法访问

### DISCUSSION THREAD

**Discussion by Thereisnoname233** [AlistGo/alist#2890] [Q&A] [answered]
从3.6.0升级到3.7.2之后，启动alist之后操作了几下之后就显示一直转圈圈，日志文件也没有相关的记录.
系统Windows10 ltsc2022

**Comment by NovaShen0724** [comment, marked as answer]
请提供相关日志和浏览器截图

**Reply by Thereisnoname233** [reply]
如图，在操作了一小会之后，再次操作就成这样了，第二张图片是在23:01时访问的，但服务端的日志却卡在22:58了
<img width="1387" alt="屏幕截图_20230103_225857" src="https://user-images.githubusercontent.com/99064124/210383674-64e360ff-6bc8-4556-b176-7c233113523d.png">
<img width="1388" alt="屏幕截图_20230103_230059" src="https://user-images.githubusercontent.com/99064124/210383688-bf109c42-0940-4f23-9fff-b48e83134b26.png">
<img width="71" alt="屏幕截图_20230103_230105" src="https://user-images.githubusercontent.com/99064124/210383691-a3828985-08b6-4593-ace0-ec3491973536.png">
<img width="838" alt="屏幕截图_20230103_230147" src="https://user-images.githubusercontent.com/99064124/210383694-3978ed4b-e8ce-4300-919d-0ec7929050b9.png">

**Reply by Thereisnoname233** [reply]
更新之前都没有这个问题的

**Reply by NovaShen0724** [reply]
没有头绪，尝试复现
请尝试将目录换成全英文

**Reply by Thereisnoname233** [reply]
我自己改了下winsw(我是靠winsw进行开机自启)的配置文件后就好了。
